orderer/consensus/etcdraft/chain_test.go

```diff
@@ -3073,6 +3073,107 @@ var _ = Describe("Chain", func() {
 			})
 
 			Context("failover", func() {
+				It("should check entries applied to raft cluster", func() {
+					By("update the snapshot interval size to 1")
+					network.stop()
+
+					network.exec(func(c *chain) {
+						c.opts.SnapshotIntervalSize = 1
+						c.opts.SnapshotCatchUpEntries = 1
+					})
+
+					By("start the network with new configuration")
+					network.init()
+					network.start()
+
+					network.stop(1)
+					network.elect(2)
+
+					// this section sends proc bype msg apt pos 0
```

Comment:
this comment is not very clear. are we trying to induce a raft msg EntryNormal with bytes 0?

Reply:
I will remove this line.